lib/handlerTemplate.js

```diff
@@ -192,7 +192,11 @@ function prepareRequestChain(conf) {
         if (requestConf.request) {
             template = new Template(requestConf.request);
             requestSpec.request = function(restbase, context) {
-                var req = template.eval(context);
+                var req = template.expand(context);
+                if (!requestConf.request.method) {
+                    // Do we really want this?
+                    req.method = context.request.method || 'get';
```

Comment:
On second thought, let's make `GET` the default if the method is not specified, regardless of the scope's method. The less magic, the better; there's already too much logic and too many things to keep track of in the config. It's risking to bite us (you know where) if we keep introducing mystic stuff  :P